install.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+DIRECTORY="fcc-python-vagrant"
+
+echo "This script requires superuser access to install apt packages."
+echo "You will be prompted for your password by sudo."
+
+# clear any previous sudo permission
+sudo -k
+
+# run inside sudo
+sudo sh <<SCRIPT
+
+echo "Installing VirtualBox ..."
+
+if hash virtualbox 2>/dev/null; then
+	echo "Virtualbox already installed, skipping ..."
+else
+	wget -q https://www.virtualbox.org/download/oracle_vbox_2016.asc -O- | apt-key add -
+	wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | apt-key add -
+
+	apt-get update -y
+	apt-get install -y virtualbox
+fi
+
+echo "Installing dkms ..."
+if hash dkms 2>/dev/null; then
+	echo "Dkms already installed, skipping ..."
+else
+	apt-get install -y dkms
+fi
+
+echo "Installing VirtualBox extensions ..."
+
+wget -O /tmp/Linux_amd64.run http://download.virtualbox.org/virtualbox/5.0.20/VirtualBox-5.0.20-106931-Linux_amd64.run
+
+chmod +x /tmp/Linux_amd64.run
+./tmp/Linux_amd64.run
```

Comment:
yes the extension pack. the runner itself figures out which version to install... i downloaded it from Oracles page only

Sent from Outlook Mobile<https://aka.ms/sdimjr>




On Sat, May 14, 2016 at 3:24 AM -0700, "Arijit Layek" <notifications@github.com<mailto:notifications@github.com>> wrote:


In install.sh<https://github.com/byteknacker/fcc-python-vagrant/pull/21#discussion_r63274127>:

> +     apt-get install -y virtualbox
> +fi
> +
> +echo "Installing dkms ..."
> +if hash dkms 2>/dev/null; then
> +     echo "Dkms already installed, skipping ..."
> +else
> +     apt-get install -y dkms
> +fi
> +
> +echo "Installing VirtualBox extensions ..."
> +
> +wget -O /tmp/Linux_amd64.run http://download.virtualbox.org/virtualbox/5.0.20/VirtualBox-5.0.20-106931-Linux_amd64.run
> +
> +chmod +x /tmp/Linux_amd64.run
> +./tmp/Linux_amd64.run


I ran the script once after removing .

Here's the screenshot

[screenshot from 2016-05-14 15 51 57]<https://cloud.githubusercontent.com/assets/5607371/15267901/0fd7148a-19ec-11e6-80bd-2542ac14eb6e.png>

Notice the error

/tmp/Linux_amd64.run: 311: exec: -title: not found

What does that install? The extension pack?

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/byteknacker/fcc-python-vagrant/pull/21/files/e00b929e46fb407e9968521c74a1487252eb6d7d#r63274127>